proposals/csharp-9.0/native-integers.md

```diff
@@ -218,6 +218,18 @@ class B : A
 }
 ```
 
+### Binary numeric promotions
+The _binary numeric promotions_ informative text (see [§11.4.7.3](https://github.com/dotnet/csharpstandard/blob/draft-v6/standard/expressions.md#11473-binary-numeric-promotions)) in C# spec) is updated as follows:
+
+> -   …
+> -   Otherwise, if either operand is of type `ulong`, the other operand is converted to type `ulong`, or a binding-time error occurs if the other operand is of `type sbyte`, `short`, `int`, **`nint`**, or `long`.
+> -   **Otherwise, if either operand is of type `nuint`, the other operand is converted to type `nuint`, or a binding-time error occurs if the other operand is of `type sbyte`, `short`, `int`, `nint`, or `long`.**
+> -   Otherwise, if either operand is of type `long`, the other operand is converted to type `long`.
+> -   Otherwise, if either operand is of type `uint` and the other operand is of type `sbyte`, `short`, **`nint`,** or `int`, both operands are converted to type `long`.
+> -   Otherwise, if either operand is of type `uint`, the other operand is converted to type `uint`.
+> -   **Otherwise, if either operand is of type `nint`, the other operand is converted to type `nint`.**
```

Comment:
How did you decide to place this bullet second from last? It's not entirely clear to me what the hierarchy should be for native integers.